dockers/docker-orchagent-cavm/port_config_cavm.ini

```diff
@@ -0,0 +1,33 @@
+# alias          lanes
```

Comment:
can we move this file to docker-orchagent/ and then use onie_platform to decide with port_config.ini to load?